app/views/users/profile.html.erb

```diff
@@ -1 +1 @@
-<%= react_component('ProfilePage', { currentUser: @current_user }) %>
```

Comment:
For profile page, let's leave it as it used to be since the `current_user` is special.